console/src/palette/Palette.tsx

```diff
@@ -61,11 +61,13 @@ export interface PaletteProps {
 type Entry = Command | ontology.Resource;
 type Key = string;
 
-const useDropOutside = ({ onDrop, canDrop, key, type }: Haul.UseDropOutsideProps) => {
+const macosUseDropOutside = ({ onDrop, canDrop, key, type }: Haul.UseDropOutsideProps) => {
```

Comment:
Could we call this function `useDropOutsideMacOS` instead of `macosUseDropOutside`? I know "use" is still in the name but I think we should explicitly start all hooks with use.